documentation/testing-docs/using-azure-test-framework.md

```diff
@@ -127,27 +135,26 @@ AZURE_TEST_MODE=Playback
+  - The Test Framework will use it to build the connection string, ignoring any settings from environment variables.
 - If JSON config file does not exist
-  - Test framework will first retrieve the environment variable `TEST_CSM_ORGID_AUTHENTICATION` and use its value to build the connection string except for the test mode (Record/Playback).
-  - Then test framework will try to get the value of the environment variable `AZURE_TEST_MODE`
-    - If `AZURE_TEST_MODE` is set, its value will be used as the test mode
-    - Otherwise, the property named `HttpRecorderMode` configured in `TEST_CSM_ORGID_AUTHENTICATION` will be used
-    - If the property `HttpRecorderMode` is also not set, `Playback` will be applied as the default value
+  - The Test Framework will first check the TEST_CSM_ORGID_AUTHENTICATION environment variable and use its value to build the connection string.
+  - If `HttpRecorderMode` is not set, then the Test Framework will check the `AZURE_TEST_MODE` environment variable.
+    - If `AZURE_TEST_MODE` is set, its value will be used as the test mode.
+    - If `AZURE_TEST_MODE` is not set, the default test mode will be `Playback`.
 
-If you are not sure the settings on your machine, please run command `Get-TestFxEnvironment`. It will consolidate above steps and display the ultimate result.
+If you are unsure about the settings on your machine, you can run the command `Get-TestFxEnvironment` to consolidate the steps mentioned above and display the final result.
 
 ## Record or Playback Tests
 
-- [Run the tests](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/azure-powershell-developer-guide.md#recordingrunning-tests) and make sure that you got a generated `.json` file that matches the test name under the `SessionRecords` folder in the test project.
+- [Run the tests](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/azure-powershell-developer-guide.md#recordingrunning-tests) and make sure that you have a generated `.json` file that corresponds to the test name, and it should be located under the `SessionRecords` folder within the test project.
 - If you want to switch from Record to Playback or from Playback to Record, consider below steps.
-  - If you opt for JSON config file, update the value of the property `HttpRecorderMode` in the JSON.
-  - If you opt for environment variables
-    - If you have `AZURE_TEST_MODE` set, update the value of this variable
-    - Otherwise, update the value of the property `HttpRecorderMode` defined in the variable `TEST_CSM_ORGID_AUTHENTICATION`
+  - If you choose to use a JSON config file, update the value of the property `HttpRecorderMode` in the JSON file.
+  - If you prefer environment variables
+    - If you have `HttpRecorderMode` defined in the environment variable `TEST_CSM_ORGID_AUTHENTICATION`, update its value accordingly.
+    - Otherwise, update the value of `AZURE_TEST_MODE`.
```

Comment:
Do we optimize these 2 options? I see that you reverse their order, but it seems the second one here is simpler.

Reply:
`AZURE_TEST_MODE` is a redundant option. I cannot remove it due to compatibility issues. But `HttpRecorderMode` in either `testcredentials.json` or environment variable `TEST_CSM_ORGID_AUTHENTICATION` has higher priority than `AZURE_TEST_MODE`.